### DISCUSSION TITLE
Auto start quest for party members

### DISCUSSION THREAD

**Discussion by pyrobane** [bumbleshoot/automate-habitica#37] [Suggestions]
Can an option be added to auto start your quest after x hours if you are the party member that invited the party to the quest but are not the party leader? It's doable on the website and app manually but getting a timer added here would be great.

**Comment by bumbleshoot** [comment]
Thanks for the suggestion @pyrobane. Why not have the party leader enable this automation instead? Then all the party's quests will be started after a certain number of hours, and each member of the party does not have to install the script/enable this automation/set it to the same number of hours.

**Reply by pyrobane** [reply]
That would be great and I'll make the suggestion. That said, my party is a bit more laid back and only a couple besides me use any form of scripting or automation. I'm just trying to do as much as I can without being dependent on the party leader to keep up with things. I did find the Auto Force Start Quests script and am testing it in conjunction with this one.
